hfaerber/Palette_Picker
# React setup

### What's this PR do? 

- Sets up the basic file structure and adds any needed dependencies. 
- Sets up the `index.js` file and the basic `App` component with their basic imports and structure including router.
- Adds `src/variables.scss` and created variables.
- Adds an images directory with our background images and a folder (`/icons`) to hold any svg icons used in the wireframe. 

----------

### How should this be manually tested? 

- From within the app's root folder: `git fetch origin` => `git checkout react-setup`.
- `npm i` || `npm install`.
- Make sure you have nothing else running on port 3000 => `npm start`.
- Navigate to `localhost:3000` in your browser. A blank black page should render. 
- Look over the files to see if you can notice anything I forgot. 

----------

### Relevant Tickets:

- This solves #18 

----------

### Screenshots: 

<details>

<summary>App in browser</summary>

 ![ ](https://user-images.githubusercontent.com/47998896/73790683-19964d80-475e-11ea-923f-e5e7985e5822.png)

</details>

----------

### Dependencies: 

- Enzyme
- Enzyme Adapter
- Enzyme-to-json
- Sass/node sass
- Prop.types
- React router

Run `npm i` || `npm install` to make sure you have all dependencies. 

----------

### Notes:

- Next steps for the FE will be to figure out our React architecture (using only hooks).
- I set up the Login component just to render something and make sure Router was working. It's currently a class component and will need to be changed when we get to #19
